### ISSUE TITLE
assert: Expected indention for a prefixed line

### ISSUE THREAD

**Issue by joeduffy** [pulumi/pulumi#1827]
I was just running an update, and received:

```
panic: fatal: An assertion has failed: Expected indention for a prefixed line

goroutine 209 [running]:
github.com/pulumi/pulumi/pkg/util/contract.failfast(0xc4213249c0, 0x3f)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/util/contract/failfast.go:23 +0xf3
github.com/pulumi/pulumi/pkg/util/contract.Assertf(0xc420298500, 0x1b01d03, 0x26, 0x0, 0x0, 0x0)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/util/contract/assert.go:33 +0x13d
github.com/pulumi/pulumi/pkg/engine.getIndentationString(0x0, 0x1ae0b2a, 0x6, 0x1, 0x3, 0xc42122f7e0)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:72 +0x1a6
github.com/pulumi/pulumi/pkg/engine.writeWithIndent(0xc4202984d0, 0x0, 0x1ae0b2a, 0x6, 0x1, 0xc42122f7e0, 0x6, 0xc42122f818, 0x1, 0x1)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:94 +0x86
github.com/pulumi/pulumi/pkg/engine.printPropertyTitle(0xc4202984d0, 0xc420860fd8, 0x3, 0x3, 0x0, 0x1ae0b2a, 0x6, 0xc42130a501)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:319 +0x186
github.com/pulumi/pulumi/pkg/engine.printObjectPropertyDiff.func1(0x1ae0b2a, 0x6, 0x10c9301)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:451 +0x6b
github.com/pulumi/pulumi/pkg/engine.printAdd(0xc4202984d0, 0x1953000, 0xc4208c9710, 0xc42122f960, 0xc420f31400, 0x0, 0x0)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:583 +0x44
github.com/pulumi/pulumi/pkg/engine.printObjectPropertyDiff(0xc4202984d0, 0xc420860fd8, 0x3, 0x3, 0xc420f2d620, 0xc420f2d650, 0xc420f2d680, 0xc420f2d6b0, 0x0, 0x0, ...)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:454 +0x139
github.com/pulumi/pulumi/pkg/engine.GetResourceOutputsPropertiesString(0x1adf08b, 0x4, 0xc4213185a0, 0x49, 0xc42114f120, 0x13, 0xc4202938f0, 0xc420293960, 0xc4202939d0, 0x0, ...)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/engine/diff.go:275 +0x36e
github.com/pulumi/pulumi/pkg/backend/local.(*ProgressDisplay).processEndSteps(0xc421266500)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/backend/local/progress.go:709 +0x997
github.com/pulumi/pulumi/pkg/backend/local.(*ProgressDisplay).processEvents(0xc421266500, 0xc421181d00, 0xc4210cd620)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/backend/local/progress.go:961 +0x18a
github.com/pulumi/pulumi/pkg/backend/local.DisplayProgressEvents.func1(0xc421266500, 0xc421181d00, 0xc4210cd620, 0xc4210cd680)
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/backend/local/progress.go:264 +0x3f
created by github.com/pulumi/pulumi/pkg/backend/local.DisplayProgressEvents
	/Users/joeduffy/dev/code/src/github.com/pulumi/pulumi/pkg/backend/local/progress.go:263 +0x4c6
```

The program is very simple:

```
import * as pulumi from "@pulumi/pulumi";
import * as gcp from "@pulumi/gcp";

const config = new pulumi.Config();

const username = "gkeadmin";
const password = config.require("clusterPassword");
const cluster = new gcp.container.Cluster("gke-cluster", {
    initialNodeCount: 3,
    nodeVersion: "latest",
    minMasterVersion: "latest",
    masterAuth: { username, password },
    nodeConfig: {
        oauthScopes: [
            "https://www.googleapis.com/auth/compute",
            "https://www.googleapis.com/auth/devstorage.read_only",
            "https://www.googleapis.com/auth/logging.write",
            "https://www.googleapis.com/auth/monitoring"
        ],
    },
});
```

FWIW, I've noticed indention is off in a few demos I've seen recently, so I'm not sure if it's related.

**Comment by lukehoban** [comment]
Dupe of #1821 - fixed with #1822?

**Comment by joeduffy** [comment]
That did the trick!
